pkg/volume/cinder/attacher.go

```diff
@@ -185,6 +186,14 @@ func (attacher *cinderDiskAttacher) VolumesAreAttached(specs []*volume.Spec, nod
 	}
 
 	instanceID, err := attacher.nodeInstanceID(nodeName)
+	// If node does not exist, node's volumes will be detached automatically and become available.
+	// So mark them detached.
+	if err == cloudprovider.InstanceNotFound {
+		for _, spec := range specs {
+			volumesAttachedCheck[spec] = false
```

Comment:
Oh, I see. Thank you very much @rootfs.
I update it soon, please take a look.
And I think the change should be inside of openstach_volume.go DiskIsAttached function too. @jingxu97